lms/djangoapps/courseware/toggles.py

```diff
@@ -78,7 +76,7 @@
 # .. toggle_use_cases: temporary, open_edx
 # .. toggle_creation_date: 2020-10-07
 # .. toggle_target_removal_date: None
-# .. toggle_warnings: Also set settings.LEARNING_MICROFRONTEND_URL and ENABLE_COURSEWARE_MICROFRONTEND and
+# .. toggle_warnings: Also set settings.LEARNING_MICROFRONTEND_URL.
```

Comment:
Nit: the "and" part of this comment got cut off, making it a bit confusing with the next line.